src/design-library/service-unavailable/service-shut-down/index.njk

```diff
@@ -0,0 +1,20 @@
+---
+layout: layout-example.njk
+---
+<h1 class="govuk-heading-xl">Sorry, the service is unavailable</h1>
+
+<p class="govuk-body">The submit an employment intermediary report service has closed.</p>
+
+<p class="govuk-body">Contact us if you need to speak to someone about the service and your reports.</p>
+
+<p class="govuk-body">
```

Comment:
Not 100% on what you mean here. If you mean: should one of the service unavailable patterns use the summary list then the answer is no, as that would substantially change the look and feel of hte pattern. 

Otherwise, can you clarify @matcarey?